README.md

```
@@ -61,18 +61,26 @@
 ### Prerequisites
 
-- Python 3.8 to 3.13
+- Python 3.11 to 3.13
+- [uv](https://docs.astral.sh/uv/) (recommended) or pip
 
 ### Steps
 
 1. **Install the Package**
 
+   Using pip (traditional method):
    ```sh
```

markdownlint-cli2 (0.17.2)

Fenced code blocks should be surrounded by blank lines
null

(MD031, blanks-around-fences)